packages/lumx-react/src/components/text-field/TextField.tsx

```diff
@@ -23,9 +23,15 @@ interface ITextFieldProps extends IGenericProps {
     /** A Chip Group to be rendered before the main text input */
     chips?: HTMLElement | ReactNode;
 
+    /**  */
+    error?: ReactNode;
```

Comment:
I would maybe put `string | ReactNode` as a type. Even though `string` is already part of the union type `ReactNode`.
I think it would encourage the user to use a string by default.
Also a JSdoc string would be nice ^^
PS: the same goes for the `helper` prop